### PR TITLE
View buildings

### PR DESCRIPTION
**Pull request by ranasobeid95**
#38

### CODE REVIEW

File: client/src/components/pages/ViewBuildings/Map.js [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+      className={styles.map}
+      maxBounds={[
+        { lat: 56.25501647203477, lng: -16.853027343750004 },
+        { lat: 49.83994655735523, lng: -17.072753906250004 },
+        { lat: 49.85410987531622, lng: 10.458984375000002 },
+        { lat: 56.30379004315596, lng: 10.72265625 },
+      ]}
+    >
+      <TileLayer
+        attribution='&copy; <a href="http://osm.org/copyright%22%3EOpenStreetMap</a> contributors'
```
Comment (FadiAlamassi) [inline client/src/components/pages/ViewBuildings/Map.js]: You can remove the attribute 
```suggestion
```

File: client/src/components/pages/ViewBuildings/Map.js [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+        url="https://api.tiles.mapbox.com/v4/{id}/{z}/{x}/{y}.png?access_token={accessToken}"
+        id="mapbox.streets"
+        accessToken="pk.eyJ1IjoiZmFkeW1hemVuIiwiYSI6ImNrMXRxY3JjZDBpMGYzbXF5YmV0c2g5a24ifQ.C6ZZQF61IMwDVQmu7Xxpzg"
```
Comment (FadiAlamassi) [inline client/src/components/pages/ViewBuildings/Map.js]: It should be used from the environment variable

File: client/src/components/pages/ViewBuildings/Map.js [hunk 3]
```diff
@@ -0,0 +1,129 @@
...
+                    <Tag className={styles.ownerLocal__active}>Local Owner</Tag>
+                  ) : (
+                    ''
+                  )}
+                </span>
+              </h2>
+
+              <h4>
+                <span className={styles.building__title}>Address:</span>
+                {address}
```
Comment (ahmedisam99) [inline client/src/components/pages/ViewBuildings/Map.js]:
```suggestion
                {address}{' '}
```

File: client/src/components/pages/ViewBuildings/Map.js [hunk 4]
```diff
@@ -0,0 +1,129 @@
...
+              </h4>
+              <h4>
+                <span className={styles.building__title}>Previous Use:</span>
+                {previousUse}
```
Comment (ahmedisam99) [inline client/src/components/pages/ViewBuildings/Map.js]:
```suggestion
                {previousUse}{' '}
```

File: client/src/components/pages/ViewBuildings/Map.js [hunk 5]
```diff
@@ -0,0 +1,105 @@
...
+            <img
+              src={thumbnail}
+              alt="buildingImage"
+              className={styles.building__img}
+            />
+            <div className={styles.popup__content}>
+              <h2 className={styles.building__city}>
+                {city}
+                <span className={styles.ownerLocal}>
+                  {isOwnerLocal === 'Yes' ? (
```
Comment (ahmedisam99) [inline client/src/components/pages/ViewBuildings/Map.js]:
```suggestion
                  {isOwnerLocal.toLowerCase() === 'Yes' ? (
```

File: client/src/components/pages/ViewBuildings/Map.js [hunk 6]
```diff
@@ -0,0 +1,105 @@
...
+                    <Tag className={styles.ownerLocal__active}>Local Owner</Tag>
+                  ) : (
+                    ''
```
Comment (ahmedisam99) [inline client/src/components/pages/ViewBuildings/Map.js]:
```suggestion
                    null
```

File: client/src/components/pages/ViewBuildings/Map.js [hunk 7]
```diff
@@ -0,0 +1,105 @@
+import React from 'react';
+import { Map, TileLayer, Marker, Popup } from 'react-leaflet';
+import PropTypes from 'prop-types';
+import { Tag } from 'antd';
+import styles from './view.module.css';
+
+const MapComponent = props => {
```
Comment (ahmedisam99) [inline client/src/components/pages/ViewBuildings/Map.js]:
```suggestion
const MapComponent = ({ buildingInfo }) => {
```

File: client/src/components/pages/ViewBuildings/Map.js [hunk 8]
```diff
@@ -0,0 +1,105 @@
...
+  const { buildingInfo } = props;
```
Comment (ahmedisam99) [inline client/src/components/pages/ViewBuildings/Map.js]:
```suggestion
  
```

File: client/src/components/pages/ViewBuildings/Map.js [hunk 9]
```diff
@@ -0,0 +1,105 @@
...
+                  )}
+                </span>
+              </h4>
+            </div>
+          </Popup>
+        </Marker>
+      );
+    });
+    return markers;
+  };
```
Comment (ahmedisam99) [inline client/src/components/pages/ViewBuildings/Map.js]: define this function outside of the component, above it maybe

File: client/src/components/pages/ViewBuildings/Map.js [hunk 10]
```diff
@@ -0,0 +1,107 @@
...
+        emptyPeriod,
+        isOwnerLocal,
+        approved,
+      } = element;
+      return (
+        <Marker position={position} key={id}>
+          <Popup>
+            <img
+              src={thumbnail}
+              alt="buildingImage"
```
Comment (ahmedisam99) [inline client/src/components/pages/ViewBuildings/Map.js]:
```suggestion
              alt="building"
```

**Pull request merged**